packages/assets/generate.ts

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
+const path = require('path');
+
+interface Entry {
+  name: string;
+  isDirectory: () => boolean;
+}
+
+const getFiles = async (entry: Entry) => {
+  const originalEntries = await fs.promises.readdir(entry, {
+    withFileTypes: true,
+  });
+
+  const files = await Promise.all(
+    originalEntries.map((foundEntry: Entry) => {
+      const res = path.resolve(entry, foundEntry.name);
+      const result = foundEntry.isDirectory() ? getFiles(res) : res;
+      return result;
+    }),
+  );
+
+  return Array.prototype.concat(...files);
+};
+
+const assets = async (orgId: string, destinationDirectory: string) => {
+  console.log(`Copying assets for ${orgId} to ${destinationDirectory}`);
+
+  const vaildOrgIds = ['atb', 'nfk'];
```

Comment:
Ikke kritisk for nå, men dette kunne til og med ha blitt hentet ut av filsystemet slik at det er enklere å utvide implementasjonen til andre orgs. Da er det bare å opprette ny mappe fremfor å lete etter "hm, hvorfor fungerer ikke dette da", om du skjønner hva jeg mener?

Reply:
Ikke kritisk for nå nei. Har som et avhjelpende tiltak eksportert `validOrgIds` fra `generate.ts` og brukt denne i `bin.ts` slik at verdiene bare trengs endres et sted i koden. Gjør det for så vidt litt enklere å legge til en dynamisk funksjon basert på `fs.readdir` også hvis OOS skulle vokse i størrelse.